js/memory.js

```diff
@@ -1,9 +1,49 @@
+// console.log("I'm running")
+
 class MemoryGame {
   constructor(cards){
     this.cards = cards;
-    // add the rest of the class properties here
+    this.pickedCards = [];
+    this.pairsClicked = 0;
+    this.pairsGuessed = 0;
+    this.score = 0;
   }
-  shuffleCards() {}
-  checkIfPair(card1, card2) {}
-  isFinished() {}
-}
\ No newline at end of file
+
+  // I've found this on stackoverflow // for now
+  shuffleCards(cards) {
+    let currentIndex = cards.length, temporaryValue, randomIndex;
```

Comment:
I would always spread these variable declarations vertically, one line for each variable. I would consider using a variable keyword for each declared variable.